### DISCUSSION TITLE
version 3.6.0 and scala-udf-similarity jar

### DISCUSSION THREAD

**Discussion by reyvasquez-vh** [moj-analytical-services/splink#1027] [Q&A]
I tried 3.6.0 today and it looks like the previous way of importing scala-udf-similarity jar is not matching with jars installed.  I'm using pyspark on AWS EMR cluster.

```
from splink.spark.jar_location import similarity_jar_location
path = similarity_jar_location()
```
```path``` points to:  ```splink/files/spark_jars/scala-udf-similarity-0.1.0_legacy.jar```
however, what is installed are:
```scala-udf-similarity-0.1.0_classic.jar``` and 
```scala-udf-similarity-0.1.0_spark3.3.jar```

In 3.4.5, returned value and installed is ```splink/files/spark_jars/scala-udf-similarity-0.0.9.jar```

**Comment by RobinL** [comment]
This looks like a bug - I've created an issue #1092
